src/parser/scanner.lex

```diff
@@ -334,7 +335,9 @@ IP_OCTET                    ([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])
                             }
 
 {DEC}*\.{DEC}+              |
-{DEC}+\.{DEC}*              {
+{DEC}+\.{DEC}*              |
+{DEC}+\.{DEC}*{EXP}         |
+{DEC}+{EXP}                 {
                                 return parseDouble();
```

Comment:
How about integer type?

Reply:
> How about integer type?

1e5 is parsed as float, not an integer.
![image](https://user-images.githubusercontent.com/38901892/107005567-d59fb380-67ca-11eb-8d1e-2338b0b97093.png)

Reply:
`2e3`?